src/main/java/net/dzikoysk/funnyguilds/command/user/CreateCommand.java

```diff
@@ -143,36 +145,35 @@ public void execute(PluginConfiguration config, MessageConfiguration messages, P
             return;
         }
 
+        Guild guild = new Guild(name);
+        guild.setTag(tag);
+        guild.setOwner(user);
+        guild.setLives(config.warLives);
+        guild.setBorn(System.currentTimeMillis());
+        guild.setValidity(System.currentTimeMillis() + config.validityStart);
+        guild.setAttacked(System.currentTimeMillis() - config.warWait + config.warProtection);
+        guild.setPvP(config.damageGuild);
+        guild.setHome(guildLocation);
+
         if (config.regionsEnabled) {
-            when (RegionUtils.isIn(guildLocation), messages.createIsNear);
-            when (RegionUtils.isNear(guildLocation), messages.createIsNear);
+            Region region = new Region(guild, guildLocation, config.regionSize);
 
-            if (config.createMinDistanceFromBorder != -1) {
+            if (config.createMinDistanceFromBorder > 0) {
```

Comment:
Assuming createMinDistanceFromBorder=0, one may expect guilds would not be allowed to be created overlapping the border, which is not the case with this implementation. Furthermore, it seems check should be active with any non-negative value and should not be allowed to be disabled. Regions shouldn't be overlapping the border at any point, this behavior clearly is always not intended and should be recognized as incompatibility with minecraft's border, because it defeats the purpose of setting a border in the first place.